services/harmony/app/routers/router.ts

```diff
@@ -289,6 +290,7 @@ export default function router({ skipEarthdataLogin = 'false' }: RouterConfig):
   result.get('/service-image-tag', asyncHandler(getServiceImageTags));
   result.get('/service-image-tag/:service', asyncHandler(getServiceImageTag));
   result.put('/service-image-tag/:service', jsonParser, asyncHandler(updateServiceImageTag));
+  result.post('/deployment-callback', jsonParser, asyncHandler(getSnsMessage));
```

Comment:
Might it make more sense for this callback route to be on the back-end instead of the front-end?